src/main/java/seedu/nutrigoals/model/TipList.java

```diff
@@ -0,0 +1,56 @@
+package seedu.nutrigoals.model;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+/**
+ * TipList class to store all tips and the random tip generator.
+ * Meant for developer use as a built-in repository of healthy-lifestyle
+ * tips.
+ */
+public class TipList {
+    private final ArrayList<String> listOfTips = new ArrayList<>();
+    private final Random r = new Random();
+
+    /**
+     * Constructs a {@code TipList} with a default list of tips.
+     */
+    public TipList() {
```

Comment:
I like the huge variety of NUS-specific tips you've written here! :+1: 

But would it be better to remove the newline characters in each of the tip strings? The size of the display text box changes if the user resizes the window. So maybe to prevent the tip from appearing as 1 long line of text, an alternative we could implement in the future is to rely on JavaFX to wrap the text within the width of the window?